manifests/README.md

```diff
@@ -0,0 +1,14 @@
+A number of manifests which will be deployed during installation.
+
+Assumption is that the list of sorted manifests is applicable using:
```

Comment:
Do you plan to hook in calling to ```oc apply```, or what's the reason for them to not be applied as part of ```make``` process?

Reply:
This PR is exclusively about providing the manifests.
@russellb and @hardys will consume them.

Thus I don#t personallyplan to do it, but the two of them I suppose.

Reply:
The current thinking is that we'll rely on the openshift-install logic that already exists to apply manifests - the "create manifests" stage generates some manifests, and we can add some extra files there, then we run the "create ignition-configs" step and those manifests are passed to the bootstrap VM and ultimately applied to the cluster.

We could manually apply them, but the idea is where possible to closely align with the existing openshift-install workflow and practices.

Do you see any barrier to trying that approach?  If not then I'll give it a try :)